wazo_calld/plugins/calls/services.py

```diff
@@ -309,6 +309,9 @@ def make_call_from_stasis_event(event):
         call.sip_call_id = channel_variables.get('WAZO_SIP_CALL_ID')
         call.line_id = channel_variables.get('WAZO_LINE_ID')
         call.creation_time = channel.get('creationtime')
+        call.is_caller = channel_variables.get('WAZO_CHANNEL_DIRECTION')
+        call.reason_code = event['cause']
```

Comment:
Are you sure that ALL stasis events have `cause` field? If not, an error will be raised

Reply:
Technically, this method is only used for a channel destoyed event, maybe the name of the method is too generic, is it ok if i simply change the name of the method to something more specific like 'channel_destroyed_event' ? That would imply that there's always a `cause` field since it's always part of the channelDestroyed stasis event. Thanks.

Reply:
Ah no it's ok